iP/src/duke/Duke.java

```diff
@@ -0,0 +1,172 @@
+package duke;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
Perhaps group imports based on the packages and leave an empty line between groups?